packages/svelte/messages/compile-errors/template.md

```diff
@@ -192,6 +192,14 @@
 
 > %thing% is invalid inside <%parent%>
```

Comment:
I don't think we're giving enough diagnostic information here. If I saw this error message...

> `<tr>` is invalid inside `<table>`

...I would tear my hair out. I think we need some supplemental information at the very least, but it'd be better still if we just _handled_ this case. What would it take for us to do that?

Reply:
I poked a bit at it this morning but didn't get very far - opened a draft PR for visibility #12628